macosx/Eternal Lands.xcodeproj/project.pbxproj

```diff
@@ -673,15 +679,15 @@
 		8D1107320486CEB800E47090 /* Eternal Lands.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "Eternal Lands.app"; sourceTree = BUILT_PRODUCTS_DIR; };
 		D730263C244EE9FD00A6C002 /* data */ = {isa = PBXFileReference; lastKnownFileType = folder; name = data; path = "../../EL Development Assets/EL Data Files/data"; sourceTree = "<group>"; };
 		D7302641244F081F00A6C002 /* Eternal Lands.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.plist.entitlements; path = "Eternal Lands.entitlements"; sourceTree = "<group>"; };
-		D7302669244F639200A6C002 /* SDL2.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = SDL2.framework; path = ../../../Library/Frameworks/SDL2.framework; sourceTree = "<group>"; };
```

Comment:
I needed to update these framework references to use the `USER_LIBRARY_DIR` var instead of relative paths. The CI's workspace is at `~/work/Eternal-Lands/Eternal-Lands` and the frameworks live in `~/Library/Frameworks`, so it was a choice between `USER_LIBRARY_DIR` or `../../../../Library/Frameworks`. I hope this is okay.